arch/risc-v/src/esp32c3/esp32c3_spi.c

```diff
@@ -849,6 +849,9 @@ static uint32_t esp32c3_spi_poll_send(FAR struct esp32c3_spi_priv_s *priv,
 {
   uint32_t val;
 
+#ifdef CONFIG_SPI_SWAPBYTES
+  if (priv->nbits == 16) wd = __builtin_bswap16(wd);
```

Comment:
It's against the NuttX Coding standard
Please see here the alternatives:
https://nuttx.apache.org/docs/latest/contributing/coding_style.html?highlight=coding%20style#if-then-else-statement